src/utils/querySelectorHelper.js

```diff
@@ -121,12 +126,27 @@ function matchRule(element, rule) {
     }
 
     if (rule.hasOwnProperty('attrs')) {
-        if (!rule.attrs.every(attr => {
+        if (!rule.attrs.some(attr => {
+            if (!element.hasAttribute(attr.name)) {
+                return false;
+            } else if (attr.operator === undefined) {
+                return true;
+            }
+
+            let value = element.getAttribute(attr.name);
+
             switch (attr.operator) {
-                case undefined:
-                    return element.hasAttribute(attr.name);
                 case '=':
-                    return element.getAttribute(attr.name) === attr.value;
+                    return value === attr.value;
+                case '^=':
+                    return value.indexOf(attr.value) === 0;
+                case '$=':
+                    return value.indexOf(attr.value) === value.length - attr.value.length;
+                case '~=':
+                    let words = value.split(' ');
+                    return words.some(word => word === attr.value);
+                case '*=':
+                    return value.indexOf(attr.value) !== -1;
```

Comment:
missing case |=

```
Represents an element with an attribute name of attr. Its value can be exactly “value” or can begin with “value” immediately followed by “-” (U+002D). It can be used for language subcode matches.
```